commons/src/main/java/org/dhis2/commons/resources/ResourceManager.kt

```diff
@@ -59,7 +60,8 @@ class ResourceManager(val context: Context) {
     }
 
     fun parseD2Error(throwable: Throwable) =
-        D2ErrorUtils(getWrapperContext()).getErrorMessage(throwable)
+        D2ErrorUtils(getWrapperContext(), NetworkUtils(getWrapperContext()))
```

Comment:
please, inject D2ErrorUtils into the ResourceManager

Reply:
Can't be done. Current implementation of resource manager depends on the activity context to provide the correct string translations. Changing it is out of the scope of the this issue.

Reply:
OK, thanks for the clarification